core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2315,7 +2315,10 @@ public static boolean hasTimeSeriesDataMap(CarbonTable carbonTable) {
     List<DataMapSchema> dataMapSchemaList = carbonTable.getTableInfo().getDataMapSchemaList();
     for (DataMapSchema dataMapSchema : dataMapSchemaList) {
       if (dataMapSchema instanceof AggregationDataMapSchema) {
-        return ((AggregationDataMapSchema) dataMapSchema).isTimeseriesDataMap();
+        boolean istimeseries = ((AggregationDataMapSchema) dataMapSchema).isTimeseriesDataMap();
```

Comment:
Please do not store isTimeseriesDatamap value in any boolean add a if check and return false